src/model/selection.js

```diff
@@ -625,6 +625,20 @@ export default class Selection {
 	}
 
 	/**
+	 * Checks whether the entire content in specified element is selected.
```

Comment:
This description is not accurate. In such case, the `<b>`'s content is also fully selected:

```
<div>[x<b>y</b>z]</div>
```

It's hard to explain this in just one paragraph, but we should try with something like:

Checks whether the selection contains the entire content of the given element. This means that selection must start at a position {@link module:engine/model/position~Position#isTouching touching} the element's start and ends at position touching the element's end.